packages/material-ui/src/locale/index.js

```diff
@@ -934,14 +1010,18 @@ export const koKR = {
 };
 
 export const nlNL = {
+  localization: {
+    formatNumber: formatNumber('nl-NL'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@JimKoene Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).